examples/history_vehicles_replacement_for_imitation_learning.py

```diff
@@ -52,17 +52,17 @@ def act(self, obs: Observation, obs_time: float) -> Tuple[float, float]:
         assert math.isclose(
             cur_state.heading, exp["heading"], abs_tol=1e-09
         ), f'vid={self._vehicle_id}: {cur_state.heading} != {exp["heading"]} @ {obs_time}'
-        # Note: the speed check can't be as tight b/c we lose some accuracy (due to angular acceleration)
+        # Note: the other checks can't be as tight b/c we lose some accuracy (due to angular acceleration)
         # by converting the acceleration vector to a scalar in the observation script,
         # which compounds over time throughout the simulation.
         assert math.isclose(
             cur_state.speed, exp["speed"], abs_tol=0.1
         ), f'vid={self._vehicle_id}: {cur_state.speed} != {exp["speed"]} @ {obs_time}'
         assert math.isclose(
-            cur_state.position[0], exp["ego_pos"][0], rel_tol=0.01
+            cur_state.position[0], exp["ego_pos"][0], abs_tol=2
```

Comment:
Note that this change, and the similar one just below are for an unrelated reason.  Basically, in testing, I discovered that when a vehicle is chosen for replacement that does a lot of non-straight driving (lane changes, etc.), position errors can compound more for it through the course of the simulation.  I'll discuss this with the Imitation learning folks, but for now, this prevents this demo code from choking.

(FWIW, the way to mitigate this is to make the Imitation action space use acceleration _vectors_ instead of an acceleration scalar, as we had yesterday.  But they asked for a scalar to reduce their model complexity.  There is already a comment about this in `observation_collection_for_imitation_learning.py`.)